src/main/java/io/github/thebusybiscuit/slimefun4/implementation/items/multiblocks/EnhancedCraftingTable.java

```diff
@@ -68,8 +67,8 @@ private void craft(Inventory inv, Block dispenser, Player p, Block b, ItemStack
         if (outputInv != null) {
             SlimefunItem sfItem = SlimefunItem.getByItem(output);
 
-            if (sfItem instanceof SlimefunBackpack) {
-                upgradeBackpack(p, inv, (SlimefunBackpack) sfItem, output);
+            if (sfItem instanceof SlimefunBackpack slimefunBackpack) {
+                upgradeBackpack(p, inv, slimefunBackpack, output);
```

Comment:
We do not always have to be so literal about it.
```suggestion
            if (sfItem instanceof SlimefunBackpack backpack) {
                upgradeBackpack(p, inv, backpack, output);
```